### ISSUE TITLE
failed to connect outlet on bootup

### ISSUE THREAD

**Issue by tlatorre-uchicago** [snoplus/orca#79]
The following message is printed to stderr when you first start ORCA:

```
2016-05-04 20:37:46.089 Orca[15244:10674893] Failed to connect (problemMatrix) outlet from (ORFec32Controller) to (NSMatrix): missing setter or instance variable
2016-05-04 20:37:51.963 Orca[15244:10674893] Failed to connect (xilinxFileField) outlet from (ORMTCController) to (NSTextField): missing setter or instance variable
```

**Comment by BenLand100** [comment]
Assuming no one is currently modifying these guis, the fix is to click the little "x" on the offending outlet in the outlets tab (right arrow in a circle) with the correct GUI element selected.

For the ORFec32Controller the offending element is the check box grid on the DB tab, and for ORMTCController it's the Label next to the Xilinx FIle... button on the settings tab.

**Comment by tlatorre-uchicago** [comment]
Damn. OK. I will look into this.

**Comment by marzece** [comment]
Maybe while we're at it should just delete the DB checkboxes . They're obviously unused and I think the plan is to implement a channel db outside of orca anyways, right?

**Comment by pershint** [comment]
Take a look at pull request #121 for a fix.  A note that the DB checkboxes matrix is still not being used anywhere; it's just linked to a defined matrix until we would want to implement it in the controller.

**Comment by tlatorre-uchicago** [comment]
Fixed in #121.
